README.md

```diff
@@ -53,19 +51,21 @@ Use in conjunction with lint and spec tests for Continuous Integration:
 
 ## Checks
 
-### hiera
+[TODO: this lists all the checks this makes, but where does it check? In the directory that the Rakefile is in, or...?]
```

Comment:
Again, this is relative to the `Rakefile`'s location. Usually the root of the module.